src/components/Widget/IntegrationsWidget.tsx

```diff
@@ -19,13 +27,14 @@ import { getProdStore } from '../../utilities/store';
 import { AsyncComponent } from '@redhat-cloud-services/frontend-components';
 import AddSourceWizard from '../addSourceWizard';
 import './IntegrationsWidget.scss';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 import { fetchCloudSources } from './services/fetchCloudSources';
 import { fetchIntegrations } from './services/fetchIntegrations';
 import { fetchRedHatSources } from './services/fetchRedHatSources';
 import { createIntegrationsData } from './consts/widgetData';
 import { useFlag } from '@unleash/proxy-client-react';
 import PermissionsChecker from '../PermissionsChecker';
+import { DropdownToggle } from '@patternfly/react-core/deprecated';
```

Comment:
It would be nice to avoid using the deprecated DropdownToggle here, to ease a bit the upgrade to PF6 that I think will remove the deprecated component completely

Reply:
This will be an issue during PF6 migration. This component no longer exists in PF6